zap/src/main/java/org/zaproxy/zap/extension/brk/BreakPanel.java

```diff
@@ -571,11 +573,15 @@ public void cont() {
         breakToolbarFactory.setBreakAll(false);
         breakToolbarFactory.setBreakRequest(false);
         breakToolbarFactory.setBreakResponse(false);
+        // Ensure Always on Top is cleared when leaving a breakpoint
+        breakpointLeft();
     }
 
     @Override
     public void drop() {
         breakToolbarFactory.drop();
+        // Ensure Always on Top is cleared when leaving a breakpoint
+        breakpointLeft();
```

Comment:
For the record, the "always on top" state is reset by the calls to `clearAndDisableRequestEventHandler´ and `clearAndDisableResponseEventHandler` which are triggered when the request/response is cleared from the GUI.